client/blocks/inline-help/admin-sections.js

```diff
@@ -0,0 +1,381 @@
+		icon: 'cog',
+	},
+	{
+		title: translate( "Delete a site or a site's content" ),
+		description: translate( 'Remove all posts, pages, and media, or delete a site completely.' ),
+		link: `/settings/general/${ siteSlug }`,
+		icon: 'cog',
+	},
+	{
+		title: translate( "Change my site's footer text" ),
+		description: translate(
+			'You can customize your website by changing the footer credit in customizer.'
+		),
+		link: `/settings/general/${ siteSlug }`,
+		synonyms: [ 'remove footer', 'update footer' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( "Export my site's content and media library" ),
+		description: translate( 'Export posts, pages and more from your site.' ),
+		link: `/settings/export/${ siteSlug }`,
+		synonyms: [ 'xml', 'images', 'migration', 'import', 'download' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( 'Manage sharing and social media connections' ),
+		description: translate(
+			'You can customize your website by changing the footer credit in customizer.'
+		),
+		link: `/sharing/${ siteSlug }`,
+		synonyms: [ 'facebook', 'twitter', 'twitter', 'tumblr', 'eventbrite' ],
+		icon: 'share',
+	},
+	{
+		title: translate( 'Add sharing buttons to my site' ),
+		description: translate(
+			'Allow readers to easily share your posts with others by adding sharing buttons throughout your site.'
+		),
+		link: `/sharing/buttons/${ siteSlug }`,
+		synonyms: [ 'like', 'reblog' ],
+		icon: 'share',
+	},
+	{
+		title: translate( 'Install, manage, and search for site Plugins' ),
+		description: translate(
+			'You can customize your website by changing the footer credit in customizer.'
+		),
+		link: `/plugins/${ siteSlug }`,
+		synonyms: [ 'upload' ],
+		icon: 'plugins',
+	},
+	{
+		title: translate( 'Approve or delete comments' ),
+		link: `/comments/all/${ siteSlug }`,
+		synonyms: [ 'spam', 'discussion', 'moderation', 'moderate' ],
+		icon: 'chat',
+	},
+	{
+		title: translate( 'Manage how users can comment on my site' ),
+		link: `/settings/discussion/${ siteSlug }`,
+		synonyms: [ 'discussion', 'moderation', 'blacklist' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( 'Manage post categories' ),
+		link: `/settings/writing/${ siteSlug }`,
+		synonyms: [ 'post', 'category' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( 'Edit my site title, tagline, or logo' ),
+		link: `/settings/general/${ siteSlug }`,
+		synonyms: [ 'title', 'logo' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( 'Set up a podcast' ),
+		link: `/settings/writing/${ siteSlug }`,
+		synonyms: [ 'podcast', 'radio', 'audio' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( "Change my site's privacy settings" ),
+		link: `/settings/general/${ siteSlug }`,
+		synonyms: [ 'privacy' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( 'Manage SEO and traffic settings' ),
+		link: `/settings/traffic/${ siteSlug }`,
+		synonyms: [ 'analytics', 'related', 'sitemap' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( 'Update my profile' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Update my profile picture' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/inline-help-add-admin-sections-ui)